### ISSUE TITLE
New room list: Bring back right click context menu on tiles

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#14260]
Open at cursor, repeated right click is possible

**Comment by t3chguy** [comment]
Yes, this was actually an accessibility thing too, tricky thing will be which menu to open. General or Notifications?

**Comment by turt2live** [comment]
For now it's deferred until we can answer that.

**Comment by t3chguy** [comment]
Given that we are saying we want a11y parity we cannot defer it.

**Comment by t3chguy** [comment]
We have consensus on the general `...` menu
